core/ir/aarch64/encode.c

```diff
@@ -205,7 +205,7 @@ const reg_id_t d_r_reg_id_to_virtual[] = {
     DR_REG_NULL, /* DR_REG_NULL */
     DR_REG_NULL, /* DR_REG_NULL */
 
-#define VIRTUAL_XREGS
+#define VIRTUAL_XREGS                                                                \
```

Comment:
I can't believe I spent a full day chasing weird bugs for two '/'.

Reply:
This happens to all of us at times :)

Reply:
:)